docs/i18n/ko/docusaurus-plugin-content-docs/current/basic-tutorial/atoms.mdx

```diff
@@ -40,7 +40,7 @@ function TodoList() {
 ```jsx
 function TodoItemCreator() {
   const [inputValue, setInputValue] = useState('');
-  const setTodoList = useSetRecoilState(todoListState);
+  const [todoList, setTodoList] = useSetRecoilState(todoListState);
```

Comment:
Part of the point of this section is to demonstrate `useSetRecoilState()` which can be used to set an atom without reading it's state or subscribing the component to re-render when it changes.

Reply:
I understood! I must have gotten confused with the `useRecoilState()` hook.